waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3884 @@
+"Kasereck",Kasereck,AT,4714.283N,01346.316E,2740.0m,7,,,,"Gipfel"
+"Kasofen",Kasofen,AT,4719.566N,01422.750E,1894.0m,7,,,,"Gipfel"
+"Kasten-Kopf",Kasteopf,AT,4725.983N,01028.350E,2129.0m,7,,,,"Gipfel"
+"Kastenwenden-Kogel",Kastegel,AT,4716.300N,01202.683E,2329.0m,7,,,,"Gipfel"
+"Katzenberg-Kogel",Katzegel,AT,4709.100N,01223.933E,3023.0m,7,,,,"Gipfel"
+"Katzen-Kogel",Katzegel,AT,4716.416N,01159.116E,2535.0m,7,,,,"Gipfel"
+"Keesegg",Keesegg,AT,4658.166N,01214.483E,3173.0m,7,,,,"Gipfel"
+"Keeskogel",Keeskgel,AT,4708.199N,01218.783E,3291.0m,7,,,,"Gipfel"
+"Keeskogel",Keeskgel,AT,4705.750N,01316.650E,2884.0m,7,,,,"Gipfel Ankogel"
+"Kegelstein",Kegelein,AT,4658.083N,01235.716E,2417.0m,7,,,,"Gipfel"
+"Keilbach-Spitze",Keilbtze,AT,4701.983N,01156.783E,3093.0m,7,,,,"Gipfel"
+"Kellerjoch",Kelleoch,AT,4719.050N,01145.933E,2237.0m,7,,,,"Gipfel"
+"Keller-Kogel",Kellegel,AT,4704.316N,01149.516E,2722.0m,7,,,,"Gipfel"
+"Kelle-Spitze",Kelletze,AT,4730.050N,01037.816E,2240.0m,7,,,,"Gipfel"
+"Kendl-Spitze",Kendltze,AT,4702.283N,01235.550E,3088.0m,7,,,,"Gipfel"
+"Kerach-Spitze",Keractze,AT,4704.899N,01112.883E,2919.0m,7,,,,"Gipfel"
+"Kessel-Alp",KesseAlp,AT,4719.400N,01228.133E,2125.0m,7,,,,"Gipfel"
+"Kesseleck",Kesseeck,AT,4720.650N,01432.383E,2314.0m,7,,,,"Gipfel"
+"Kesselkofel",Kessefel,AT,4637.899N,01248.400E,2251.0m,7,,,,"Gipfel"
+"Kessel-Spitze",Kessetze,AT,4705.983N,01121.900E,2728.0m,7,,,,"Gipfel"
+"Kienberg",Kienberg,AT,4730.683N,01157.400E,1746.0m,7,,,,"Gipfel"
+"Kienberg SA",KienbgSA,AT,4653.217N,01438.733E,2050.0m,7,,,,"Gipfel"
+"Kienleiten-Kopf",Kienlopf,AT,4723.150N,01118.933E,1611.0m,7,,,,"Gipfel"
+"Kilnprein",Kilnpein,AT,4659.116N,01350.616E,2408.0m,7,,,,"Gipfel"
+"Kirbisch",Kirbisch,AT,4703.089N,01403.204E,2142.0m,7,,,,"Gipfel"
+"Kirchberg",Kircherg,AT,4732.233N,01232.250E,1678.0m,7,,,,"Gipfel"
+"Kirchdach-Spitze",Kirchtze,AT,4704.016N,01120.500E,2840.0m,7,,,,"Gipfel"
+"Kirchel",Kirchel,AT,4729.750N,01237.666E,1692.0m,7,,,,"Gipfel"
+"Kirchlispitzen",Kirchzen,AT,4702.952N,00946.458E,2551.0m,7,,,,"Gipfel"
+"Kitzbuehelerhorn",Kitzborn,AT,4728.599N,01225.816E,1996.0m,7,,,,"Sendemast"
+"Kitzkar-Kogel",Kitzkgel,AT,4709.166N,01236.083E,2801.0m,7,,,,"Gipfel"
+"Kitzstein",Kitzsein,AT,4717.066N,01316.216E,2037.0m,7,,,,"Gipfel"
+"Kitzsteinhorn",Kitzsorn,AT,4711.249N,01241.250E,3203.0m,7,,,,"Gipfel"
+"Kl. Buchstein",Kl.Buein,AT,4737.766N,01437.483E,1990.0m,7,,,,"Gipfel"
+"Kl. Grubenkar-Kogel",Kl.Grgel,AT,4704.466N,01312.783E,2905.0m,7,,,,"Gipfel"
+"Kl. Heissen-Kopf",Kl.Heopf,AT,4722.866N,01126.116E,2038.0m,7,,,,"Gipfel"
+"Kl. Hetzkogel",Kl.Hegel,AT,4750.383N,01503.133E,1493.0m,7,,,,"Gipfel"
+"Kl. Karkopf",Kl.Kaopf,AT,4721.350N,01101.533E,2471.0m,7,,,,"Gipfel"
+"Kl. Koenigstuhl",Kl.Koeuhl,AT,4700.433N,01348.270E,2254.0m,7,,,,"Gipfel"
+"Kl. Landeck-Kogel",Kl.Lagel,AT,4708.133N,01233.100E,2259.0m,7,,,,"Gipfel"
+"Kl. Oetscher",Kl.Oether,AT,4751.000N,01509.500E,1552.0m,7,,,,"Gipfel"
+"Kl. Rettenstein",Kl.Reein,AT,4720.166N,01220.450E,2217.0m,7,,,,"Gipfel"
```

Comment:
Abbreviation consistency especially in the short form (field2)

Reply:
I propose to remove the field 2 entries. This is what how2soar does. I don't get the added value of the field2 anyhow.